src/coreclr/src/vm/pefile.cpp

```diff
@@ -2463,22 +2463,22 @@ PTR_ICLRPrivBinder PEFile::GetBindingContext()
     return pBindingContext;
 }
 
-#ifndef DACCESS_COMPILE
-AssemblyLoadContext* PEFile::GetAssemblyLoadContext()
+PTR_AssemblyLoadContext PEFile::GetAssemblyLoadContext()
 {
```

Comment:
Would it be better to store the `PTR_AssemblyLoadContext` and the object handle in `AssemblyLoadContext` base class to avoid DACizing the whole binder machinery?

Reply:
I am worried that we will keep finding holes in the binder DACization.

Reply:
> Would it be better to store the `PTR_AssemblyLoadContext` in the PEFile and the object handle in `AssemblyLoadContext` base class to avoid DACizing the whole binder machinery?

I will try to do it that way.